data/olm-catalog/tekton-tasks-operator.clusterserviceversion.yaml

```diff
@@ -261,7 +270,7 @@ spec:
                   valueFrom:
                     fieldRef:
                       fieldPath: metadata.namespace
-                image: quay.io/kubevirt/tekton-tasks-operator:v0.1.2
+                image: quay.io/ksimon/tekton-tasks-operator:latest
```

Comment:
Hi @ksimon1 ,
Is this change intended, using your own quay account instead of kubevirt's ?

Reply:
This was very likely not intended but it is also overwritten by the csv-generator during release. We will however still revert it.